src/vstart.sh

```diff
@@ -425,6 +429,12 @@ wconf() {
 	fi
 }
 
+get_nvmeserialnumber() {
+    if [ "$spdk_enabled" -eq 1 ]; then
+        nvmeserialnumber=`sudo lspci -vvv -d 8086:0953 | grep 'Device Serial Number' | awk '{print $NF}' | sed 's/-//g'`
```

Comment:
might want to allow user to set the PCI_ID via command line options or environment variable, instead of hard-wiring it to a certain vendor or a certain product line.

no need to set `nvmeserialnumber` here, please just echo the serial number here. and use `get_nvme_serial_number` like:
```
bluestore_block_path = spdk:$(get_nvme_serial_number)"
```

Reply:
Will update soon.